package-lock.json

```diff
@@ -43255,245 +43221,89 @@
       }
     },
     "node_modules/mongodb-schema": {
-      "version": "8.2.5",
-      "resolved": "https://registry.npmjs.org/mongodb-schema/-/mongodb-schema-8.2.5.tgz",
-      "integrity": "sha512-847OmAJR5s4kUKN/ZxL3GnsddGiCaXXZ7T5b6So3yl+GTwgoTGSryJEAxkCpzaPo5NMxIF606tK16Sumnt6KBg==",
+      "version": "10.0.0",
+      "resolved": "https://registry.npmjs.org/mongodb-schema/-/mongodb-schema-10.0.0.tgz",
+      "integrity": "sha512-7dRmwwB2g3a0qUDBafADkJv36aQPXW5Cf4ycFQqiABtU43Aj2eWOMYhP6lnoqQDwc7lWMg2q/TQs7piPzDAToA==",
       "dev": true,
       "dependencies": {
-        "async": "^1.5.2",
-        "event-stream": "^4.0.1",
-        "lodash": "^3.8.0",
-        "progress": "^1.1.8",
+        "lodash": "^4.17.21",
+        "progress": "^2.0.3",
         "reservoir": "^0.1.2"
       },
       "bin": {
         "mongodb-schema": "bin/mongodb-schema"
       },
       "optionalDependencies": {
-        "cli-table": "^0.3.1",
-        "js-yaml": "^3.5.2",
-        "mongodb": "^3.1.4",
-        "mongodb-collection-sample": "^4.4.2",
-        "mongodb-extended-json": "^1.6.2",
-        "mongodb-ns": "^2.0.0",
-        "numeral": "^1.5.3",
-        "stats-lite": "^2.0.0",
-        "yargs": "^3.32.0"
-      }
-    },
-    "node_modules/mongodb-schema/node_modules/async": {
-      "version": "1.5.2",
-      "resolved": "https://registry.npmjs.org/async/-/async-1.5.2.tgz",
-      "integrity": "sha1-7GphrlZIDAw8skHJVhjiCJL5Zyo=",
-      "dev": true
-    },
-    "node_modules/mongodb-schema/node_modules/bl": {
-      "version": "2.2.1",
-      "resolved": "https://registry.npmjs.org/bl/-/bl-2.2.1.tgz",
-      "integrity": "sha512-6Pesp1w0DEX1N550i/uGV/TqucVL4AM/pgThFSN/Qq9si1/DF9aIHs1BxD8V/QU0HoeHO6cQRTAuYnLPKq1e4g==",
-      "dev": true,
-      "optional": true,
-      "dependencies": {
-        "readable-stream": "^2.3.5",
-        "safe-buffer": "^5.1.1"
-      }
-    },
-    "node_modules/mongodb-schema/node_modules/bson": {
-      "version": "1.1.6",
-      "resolved": "https://registry.npmjs.org/bson/-/bson-1.1.6.tgz",
-      "integrity": "sha512-EvVNVeGo4tHxwi8L6bPj3y3itEvStdwvvlojVxxbyYfoaxJ6keLgrTuKdyfEAszFK+H3olzBuafE0yoh0D1gdg==",
-      "dev": true,
-      "optional": true,
-      "engines": {
-        "node": ">=0.6.19"
-      }
-    },
-    "node_modules/mongodb-schema/node_modules/camelcase": {
-      "version": "2.1.1",
-      "resolved": "https://registry.npmjs.org/camelcase/-/camelcase-2.1.1.tgz",
-      "integrity": "sha1-fB0W1nmhu+WcoCys7PsBHiAfWh8=",
-      "dev": true,
-      "optional": true,
-      "engines": {
-        "node": ">=0.10.0"
+        "bson": "^5.0.1",
+        "cli-table": "^0.3.4",
+        "js-yaml": "^4.0.0",
+        "mongodb": "^4.13.0",
```

Comment:
I’m not sure if there’s an easy to way to remove unused optional dependencies of our own dependencies, but we probably don’t need another extra copy of mongodb@4.x + bson@4.x